### ISSUE TITLE
Docstring of automodule does not start a new paragraph in LaTeX, but in HTML

### ISSUE THREAD

**Issue by shimizukawa** [sphinx-doc/sphinx#556]
In the following example, the behaviour will differ between LaTeX and HTML output:

```
This is a simple test paragraph.

.. automodule:: foo
    :members: bar
```

The docstring of the module will end up in the same paragraph as the previous line in LaTeX, but not in HTML. The latter behaviour seems to be more sensible.

A workaround is to place

```
.. raw:: latex

    \\
```

before the automodule directive, but that has some issues in terms of formatting, spacing, etc. -- aside from being ugly.

---
- Bitbucket: https://bitbucket.org/birkenfeld/sphinx/issue/556
- Originally reported by: [hmgaudecker](https://bitbucket.org/hmgaudecker)
- Originally created at: 2010-11-11T11:28:09.765

**Comment by shimizukawa** [comment]
_From Georg Brandl on 2011-01-03 20:38:30+00:00_

Should be fixed with <<cset 0cda61e88d903f5c675dc0acb193ca1e861ad751>>.

**Comment by shimizukawa** [comment]
_From Georg Brandl on 2014-10-18 07:30:26+00:00_

Removing version: 1.0.4 (automated comment)
